crescent/commands/decorators.py

```diff
@@ -156,15 +159,19 @@ def command(
         defaults: Dict[str, Any] = {}
 
         options: list[CommandOption] = []
+        name_map: dict[str, str] = {}
         for n, v in callback.__dict__.items():
             if not isinstance(v, ClassCommandOption):
                 continue
-            options.append(v._gen_option(n))
-            defaults[n] = v.default
+            options.append(generated := v._gen_option(n))
+            if generated.name != n:
+                name_map[generated.name] = n
+            defaults[generated.name] = v.default
```

Comment:
I don't think the walrus operator helps readability here
```suggestion
            generated = v._gen_option(n)
            options.append(generated)

            if generated.name != n:
                name_map[generated.name] = n

            defaults[generated.name] = v.default
```